extension/logstash/logstash-output-doris.gemspec

```diff
@@ -0,0 +1,44 @@
+=begin
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+=end
+Gem::Specification.new do |s|
+  s.name            = 'logstash-output-doris'
+  s.version         = '0.1.0'
+  s.licenses        = ['Apache License (2.0)']
```

Comment:
`Apache License (2.0)` seems not a valid license name in my env.
Should it be `Apache-2.0`